src/styles/Search.css

```diff
@@ -0,0 +1,105 @@
+.Search {
+  display: flex;
+  flex-direction: column;
```

Comment:
When you have some css that is shared over lots of components, you can make a base class name and then just apply the differences to the specific id, or a further classname for anything that is unique